containers/layout/Layout.tsx

```diff
@@ -12,29 +12,27 @@ import { useAuth } from 'hooks/useAuth';
 import { useRouter } from 'next/router';
 import { FirebaseError } from 'lib/types/auth';
 import Error from 'components/error';
+import { withTranslation } from 'i18n.config';
+import { layoutVar } from 'lib/context-variables';
+import { useReactiveVar } from '@apollo/client';
 
 export interface LayoutProps {
-  home?: boolean;
-  heading?: string;
-  title?: string | string[];
-  error: FirebaseError | null;
   children?: React.ReactNode;
   i18n?: I18n;
   t: TFunction;
 }
 
-const Layout = ({
-  home = false,
-  error = null,
-  children,
-  heading,
-  title,
-  i18n,
-  t,
-}: LayoutProps): React.ReactElement => {
+const Layout = ({ children, t, i18n }: LayoutProps): React.ReactElement => {
   const classes = useStyles();
   const auth = useAuth();
   const router = useRouter();
+  const { pathname } = router;
+
+  const isHome = pathname === '/';
+
+  const { title, heading, error } = useReactiveVar(layoutVar);
```

Comment:
what happened to the idea to use your getTitleFromPathName function for title (and not have title in reactiveVar at all?

Reply:
sorry, this is just because I didn't include that in the DRAFT PR yet.